frontend/src/views/IntegrationsPage/components/IntegrationsSection/IntegrationsSection.tsx

```diff
@@ -45,15 +52,31 @@ export const IntegrationsSection = ({
           <Skeleton className="h-28" />
         </div>
       )}
-      {!isLoading && !integrations.length && (
+
+      {!bot?.isActive && (
+        <div className="px-6 py-4">
+          <Alert variant="warning">
+            <FontAwesomeIcon icon={faExclamationTriangle} />
```

Comment:
Instead of putting icons like this i would say when asking for variants automatically set it. 

Like variant warning means this particular icon. I see also there is an alignment issue. So just ask for variant and there will be also a default titile, user's can override the title when using that component then children